patches/components-content_settings-core-browser-content_settings_registry.cc.patch

```diff
@@ -1,7 +1,25 @@
 diff --git a/components/content_settings/core/browser/content_settings_registry.cc b/components/content_settings/core/browser/content_settings_registry.cc
-index 03d0c89f73cb7a6a615195ca9ee36881fd4531c2..5e686c535d879f65f5a316041e9ad595d31a4285 100644
+index 03d0c89f73cb7a6a615195ca9ee36881fd4531c2..53f772a6d757d070b09ab2ddd5dba6dd24293d9b 100644
 --- a/components/content_settings/core/browser/content_settings_registry.cc
```

Comment:
a patch already exists for this, please change the settings in `ContentSettingsRegistry::BraveInit` or better yet, clean up the existing patch by removing the patch on the header file and changing the patch in the cc file to be a macro

Reply:
I went for the macro and remove the patch for the `.h` file.